RegionalOverviews/data_preparation_RDBES/001_read_and_prepare_data_RDBES_CL_CE.R

```diff
@@ -50,34 +57,36 @@ if (!dir.exists(dir_output_rcg)){
 ## ======================== 
 ## Here we obtain raw RDBES data. 
 #  The preferable choice is to use a function downloading the data from the SharePoint. Alternatively, data are to be manually downloaded. 
-source("RegionalOverviews/funs/func_download_data_from_sharepoint.r")
-download_data_from_sharepoint(
- sharepoint_address = "Path to directory on SharePoint",
- filename_vector = paste0(target_region, ".zip"), 
- dir_download_browser = "//storage-lk.slu.se/home$/erqu0001/Downloads", # Directory where browser downloads, e.g. on eros machine
- dir_download_target = "Path to directory where data should be stored",  
- unzip=TRUE
-)
+source("RegionalOverviews/funs_RDBES/func_download_data_from_sharepoint.r")
+# download_data_from_sharepoint(
+ # sharepoint_address = "https://community.ices.dk/ExpertGroups/DataExpports/RCG/_layouts/15/start.aspx#/RCG%20Data/Forms/AllItems.aspx?View=%7BFC9DF179%2DB628%2D47C5%2DB2A4%2D1D945AB1BBE4%7D",#"Path to directory on SharePoint",
+ # filename_vector = paste0(target_region, ".zip"), 
+ # dir_download_browser = "C:/Users/acfernandes/Downloads", # Directory where browser downloads, e.g. on eros machine
+ # dir_download_target = "RegionalOverviews/data_RDBES/001_raw",#"Path to directory where data should be stored",  
+ # unzip=TRUE
+# )
 
 # ========================
 # reads in data
 # ========================
 
 # reads aux_countries dataset
-aux_countries<-read.table("RegionalOverviews//data//aux_countries.txt", sep=",", header=T, colClasses="character", na.strings = "")
-
-# reads aux_species dataset (scientific name, catch group)
-aux_species<-read.table("RegionalOverviews//data//ASFIS_WoRMS.csv", sep=",", header=T, colClasses="character", na.strings = "")
+aux_countries<-read.table("RegionalOverviews/data/aux_countries.txt", sep=",", header=T, colClasses="character", na.strings = "")
+aux_species <- read.csv("RegionalOverviews/data/ASFIS_WoRMS.csv", sep=",", header=T, colClasses="character", na.strings = "")
 
 # reads RDBES data
 RDBESdataPath = 'RegionalOverviews/data_RDBES/001_raw'
+setwd('RegionalOverviews/data_RDBES/001_raw')
+
 
 file_cl <- paste(RDBESdataPath, "/RDBES CL/CommercialLanding.csv", sep = '')
-file_ce <- paste(RDBESdataPath, "/RDBES CE/CommercialEffort.csv" , sep = '')
+file_ce <- paste(RDBESdataPath, "/RDBES CE/CommercialEffort.csv" , sep = '') 
 
 # read data
-ce<-data.table::fread(file_ce, stringsAsFactors=FALSE, verbose=FALSE, sep=",", na.strings="NULL",quote = "")
-cl<-data.table::fread(file_cl, stringsAsFactors=FALSE, verbose=FALSE, sep=",", na.strings="NULL",quote = "")
+#cl <- data.table::fread(file_cl, stringsAsFactors=FALSE, verbose=FALSE, fill=TRUE, sep=",", na.strings="NULL")
+cl <- data.table::fread(file_cl)
```

Comment:
I used the option without parameters because I was not able to read the data if they were there (R session aborting all the times...). Asked Josefine and she was able to read it this way. So I used the same so I could work with the script.
It's OK to change it for the fread with parameters if it works for you.

Reply:
stringsAsFactors=FALSE, verbose=FALSE, fill=TRUE, sep=",", na.strings="NULL" <- these parameters aborting the session, I had a problem too.

stringsAsFactors=FALSE, verbose=FALSE, sep=",", na.strings="NULL",quote = "" <- these parameters works OK